client/lib/plans/features-list.js

```diff
@@ -1317,13 +1322,38 @@ export const FEATURES_LIST = {
 		getSlug: () => FEATURE_LIVE_CHAT_SUPPORT_BUSINESS_DAYS,
 		getTitle: () => i18n.translate( 'Live chat support 24X5' ),
 		getDescription: () =>
-			i18n.translate( 'Live chat is available 24 hours a day from Monday through Friday.' ),
+			i18n.translate( 'Live chat is available 24 hours a day from Monday through Friday' ),
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.